generate_ldscript.py

```diff
@@ -128,11 +140,13 @@ def main(argv):
 
     values = {
         "memories": sorted_memories,
+        "ram_memories": sorted_ram_memories,
         "default_stack_size": "0x400",
         "default_heap_size": "0x800",
         "num_harts": len(harts),
         "boot_hart": boot_hart,
         "chicken_bit": 1,
+        "eccscrub_bit": 1,
```

Comment:
At some point we should figure out how to detect when we can turn ECC scrubbing off.

Reply:
> At some point we should figure out how to detect when we can turn ECC scrubbing off.

What I have in mind is to have a "meta, scrub = 1 | 0" in the chosen node. Which can be add later.

Reply:
Yeah, we can add that after we get this working at least.